orc8r/cloud/go/obsidian/tests/unit_test_utils.go

```diff
@@ -110,6 +111,18 @@ func GetHandlerByPathAndMethod(t *testing.T, handlers []obsidian.Handler, path s
 	return obsidian.Handler{}
 }
 
+func YAMLToJSONConverter(s string) encoding.BinaryMarshaler {
+	return &yamlToJsonConverter{yaml: s}
+}
+
+type yamlToJsonConverter struct {
+	yaml string
+}
+
+func (j *yamlToJsonConverter) MarshalBinary() (data []byte, err error) {
+	return yaml.YAMLToJSON([]byte(j.yaml))
+}
+
```

Comment:
This is a bit of an ugly workaround but SwaggerUIBundle doesn't play well with the output of the existing json.Marshal() and yaml.YAMLToJson seems to require the least amount of extra work. Resultingly, I had to settle for adding this construct to satisfy the encoding.BinaryMarshaler field of TestCase